_includes/pbc/all/install-features/202304.0/install-the-service-points-customer-account-management-feature.md

```diff
@@ -0,0 +1,163 @@
+| ServicePointCheckoutMultiShippingAddressesFormExpanderPlugin    | Expands checkout multi-shipping address form with `ServicePoint` subform. | None          | SprykerShop\Yves\ServicePointWidget\Plugin\CustomerPage    |
+| ServicePointAddressCheckoutAddressCollectionFormExpanderPlugin  | Expands shipments with service point address.                             | None          | SprykerShop\Yves\ServicePointWidget\Plugin\CustomerPage    |
+| ServicePointAddressFormWidgetCacheKeyGeneratorStrategyPlugin    | Skips caching of `ServicePointAddressFormWidget` widget.                  | None          | SprykerShop\Yves\ServicePointWidget\Plugin\ShopApplication |
+
+**src/Pyz/Yves/CustomerPage/CustomerPageDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\CustomerPage;
+
+use SprykerShop\Yves\CustomerPage\CustomerPageDependencyProvider as SprykerShopCustomerPageDependencyProvider;
+use SprykerShop\Yves\ServicePointWidget\Plugin\CustomerPage\ServicePointAddressCheckoutAddressCollectionFormExpanderPlugin;
+use SprykerShop\Yves\ServicePointWidget\Plugin\CustomerPage\ServicePointCheckoutAddressCollectionFormExpanderPlugin;
+use SprykerShop\Yves\ServicePointWidget\Plugin\CustomerPage\ServicePointCheckoutMultiShippingAddressesFormExpanderPlugin;
+
+class CustomerPageDependencyProvider extends SprykerShopCustomerPageDependencyProvider
+{
+    /**
+     * @return list<\SprykerShop\Yves\CustomerPageExtension\Dependency\Plugin\CheckoutAddressCollectionFormExpanderPluginInterface>
+     */
+    protected function getCheckoutAddressCollectionFormExpanderPlugins(): array
+    {
+        return [
+            new ServicePointCheckoutAddressCollectionFormExpanderPlugin(),
+            new ServicePointAddressCheckoutAddressCollectionFormExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return list<\SprykerShop\Yves\CustomerPageExtension\Dependency\Plugin\CheckoutMultiShippingAddressesFormExpanderPluginInterface>
+     */
+    protected function getCheckoutMultiShippingAddressesFormExpanderPlugins(): array
+    {
+        return [
+            new ServicePointCheckoutMultiShippingAddressesFormExpanderPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\ShopApplication;
+
+use SprykerShop\Yves\ServicePointWidget\Plugin\ShopApplication\ServicePointAddressFormWidgetCacheKeyGeneratorStrategyPlugin;
+use SprykerShop\Yves\ShopApplication\ShopApplicationDependencyProvider as SprykerShopApplicationDependencyProvider;
+
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+    /**
+     * @return array<\SprykerShop\Yves\ShopApplicationExtension\Dependency\Plugin\WidgetCacheKeyGeneratorStrategyPluginInterface>
+     */
+    protected function getWidgetCacheKeyGeneratorStrategyPlugins(): array
+    {
+        return [
+            new ServicePointAddressFormWidgetCacheKeyGeneratorStrategyPlugin(),
```

Comment:
* The ClickCollectServicePointAddressFormWidgetCacheKeyGeneratorStrategyPlugin is missing.